2025-02-26 17:02:45,125 ERROR: Exception during request: expected token 'end of statement block', got '[' [in /data/myblog/app/extensions.py:176]
2025-02-26 17:02:46,314 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 17:02:46,314 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 17:02:46,315 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 17:02:46,315 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 17:02:46,333 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 17:02:46,333 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 17:02:46,351 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 17:02:46,352 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 17:02:46,352 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 17:02:46,897 INFO: 请求: GET http://192.168.72.128:5000/blog/ [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:46,918 INFO: Locating template 'blog/index.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/index.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:02:46,931 INFO: Locating template 'blog/layout.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/layout.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:02:46,990 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:47,042 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:47,042 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 17:02:47,043 INFO: Locating template 'errors/404.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/errors/404.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:02:47,044 INFO: Locating template 'blog/base.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/base.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:02:48,026 INFO: 请求: GET http://192.168.72.128:5000/blog/tags [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:48,027 INFO: Locating template 'blog/tags.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/tags.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:02:48,081 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:48,110 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:48,110 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 17:02:50,335 INFO: 请求: GET http://192.168.72.128:5000/blog/archive [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:50,348 INFO: Locating template 'blog/archive.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/archive.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:02:50,403 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:50,422 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:50,423 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 17:02:51,862 INFO: 请求: GET http://192.168.72.128:5000/blog/tags [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:51,900 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:51,927 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 17:02:51,927 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
2025-02-26 17:02:58,322 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 17:02:58,323 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 17:02:58,323 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 17:02:58,323 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 17:02:58,340 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 17:02:58,340 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 17:02:58,356 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 17:02:58,356 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 17:02:58,357 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 17:03:10,186 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 17:03:10,186 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 17:03:10,187 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 17:03:10,187 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 17:03:10,205 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 17:03:10,205 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 17:03:10,221 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 17:03:10,221 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 17:03:10,222 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 17:04:02,652 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 17:04:02,653 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 17:04:02,653 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 17:04:02,653 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 17:04:02,670 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 17:04:02,670 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 17:04:02,687 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 17:04:02,687 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 17:04:02,688 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 17:04:03,262 INFO: MyBlog 启动 [in /data/myblog/app/__init__.py:69]
2025-02-26 17:04:03,262 INFO: 配置模式: development [in /data/myblog/app/__init__.py:70]
2025-02-26 17:04:03,262 INFO: 调试模式: True [in /data/myblog/app/__init__.py:71]
2025-02-26 17:04:03,263 INFO: 日志文件: /data/myblog/logs/myblog.log [in /data/myblog/app/__init__.py:72]
2025-02-26 17:04:03,278 INFO: 扩展初始化完成 [in /data/myblog/app/__init__.py:85]
2025-02-26 17:04:03,278 INFO: 数据库迁移初始化完成 [in /data/myblog/app/__init__.py:89]
2025-02-26 17:04:03,294 INFO: 蓝图注册完成 [in /data/myblog/app/__init__.py:96]
2025-02-26 17:04:03,294 INFO: 过滤器初始化完成 [in /data/myblog/app/__init__.py:115]
2025-02-26 17:04:03,295 INFO: 应用初始化完成 [in /data/myblog/app/__init__.py:166]
2025-02-26 17:04:25,017 INFO: 请求: GET http://192.168.72.128:5000/blog/tags [in /data/myblog/app/__init__.py:141]
2025-02-26 17:04:25,020 INFO: Locating template 'blog/tags.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/tags.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:04:25,026 INFO: Locating template 'blog/layout.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/layout.html') [in /home/ops/.local/lib/python3.12/site-packages/flask/debughelpers.py:178]
2025-02-26 17:04:25,063 INFO: 请求: GET http://192.168.72.128:5000/static/css/blog.css [in /data/myblog/app/__init__.py:141]
2025-02-26 17:04:25,183 INFO: 请求: GET http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:141]
2025-02-26 17:04:25,184 WARNING: 页面未找到: http://192.168.72.128:5000/favicon.ico [in /data/myblog/app/__init__.py:109]
